packages/sanity/src/core/form/inputs/InvalidValueInput/UntypedValueInput.tsx

```diff
@@ -34,17 +35,9 @@ function SetMissingTypeButton({
   return (
     <Button
       onClick={handleClick}
-      text={
-        <>
-          <Translate
```

Comment:
What's the difference in using  `<Translate />` vs `t()` ? Button is expecting a string in text and if we use the `<Translate /> ` it return a node